wordpress-mobile/WordPress-Android
# Replace the Jetpack splash icon with Wordpress+Jetpack logo

Convo: p1703363229811979-slack-C0JJ0C1UL
also:
Fixes https://github.com/Automattic/jetpack-mobile-roadmap/issues/96
PT: pcdRpT-58l-p2

This PR contains a replaced logo on the splash screen.
Additionally, I fixed a color for the bottom panel of the login screen - it also has to be white instead of light green.

| Before | After |
| -------| ----- |
|<image width="350px" src="https://github.com/wordpress-mobile/WordPress-Android/assets/16563318/47537361-1010-45e1-a2ff-9067d4341170"/>|<image width="350px" src="https://github.com/wordpress-mobile/WordPress-Android/assets/16563318/a52b3e13-0aab-47e8-baf4-491cb201dc84"/>|
|<image width="350px" src="https://github.com/wordpress-mobile/WordPress-Android/assets/16563318/7b783d27-efd7-4f55-a547-3e0e9f5ce277"/>|<image width="350px" src="https://github.com/wordpress-mobile/WordPress-Android/assets/16563318/5c9bcaa3-c020-4c22-bcb9-bf2b2c1078e2)"/>|
|<image width="350px" src="https://github.com/wordpress-mobile/WordPress-Android/assets/16563318/f7b86536-edfa-42b3-9fd9-3210a8178599"/>|<image width="350px" src="https://github.com/wordpress-mobile/WordPress-Android/assets/16563318/efb46dad-6a97-4fab-8b52-b3b7d07ec293"/>|

-----

## To Test:

- Run the app and make sure you can see an updated logo on the splash screen.
- Make sure the bottom panel with action buttons has a white color instead of ultra-light green.

-----

## Regression Notes

1. Potential unintended areas of impact

    - Splash screen, Login screen

2. What I did to test those areas of impact (or what existing automated tests I relied on)

    - Manual testing

3. What automated tests I added (or what prevented me from doing so)

    - UI changes only

-----

## PR Submission Checklist:


-----

## UI Changes Testing Checklist:
